app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -25,6 +25,7 @@ def prov_field_changed
       refresh_divs = prov_get_form_vars  # Get changed option, returns true if divs need refreshing
       build_grid if refresh_divs
       changed = (@edit[:new] != @edit[:current])
+      @tooltip = MiqTemplate.find(@edit[:new][:src_vm_id].first).try(:decorate).try(:size_tooltip)
```

Comment:
Is the `@edit[:new][:src_vm_id].first` an array? Do you know why?
Also please set just a `@record` instead of `@tooltip` and call the `decorate` and `size_tooltip` in the view.